.github/checkgroup.yml

```diff
@@ -25,27 +25,25 @@ subprojects:
       - ".actions/**"
     checks:
       # Note: updates here should be applied to the lightning_lite group
-      - "pl-cpu (macOS-11, pytorch, 3.8, 1.10)"
-      - "pl-cpu (macOS-11, pytorch, 3.9, 1.11)"
-      - "pl-cpu (macOS-11, pytorch, 3.10, 1.12)"
+      - "pl-cpu (macOS-11, pytorch, 3.8, 1.11)"
+      - "pl-cpu (macOS-11, pytorch, 3.9, 1.12)"
+      - "pl-cpu (macOS-11, pytorch, 3.10, 1.13)"
       - "pl-cpu (macOS-11, pytorch, 3.7, 1.9, oldest)"
```

Comment:
where is 1.10?

Reply:
we do not have full range for mac

Reply:
Why not? We used to test all PyTorch versions, right?

Reply:
Yes with conda so on Ubuntu as we still, right?